examples/pytorch/README.md

```diff
@@ -251,3 +252,87 @@ or if in a Conda environment:
 ```bash
 conda install -c comet_ml -c anaconda -c conda-forge comet_ml
 ```
+
+### Neptune
+
+1. Install the Neptune client library:
```

Comment:
Can we use a bit more prose here and remove the list? Something like:

First, install the Neptune client library. You can do it with `pip`: ... or `conda`...

Reply:
@normandy7, could you take a look please as an author?

Reply:
Done